packages/app/src/cli/services/function/replay.test.ts

```diff
@@ -111,6 +113,7 @@ describe('replay', () => {
 
   test('throws error if no logs available', async () => {
     // Given
+    vi.mocked(existsSync).mockReturnValue(false)
```

Comment:
Should we treat this as a distinct test case from there being no files in the directory?